app/controllers/organisations_controller.rb

```diff
@@ -16,6 +16,7 @@ def show
     @show = Organisations::ShowPresenter.new(@organisation)
     @header = Organisations::HeaderPresenter.new(@organisation)
     @documents = Organisations::DocumentsPresenter.new(@organisation)
+    @supergroups = Organisations::SupergroupsPresenter.new(@organisation)
```

Comment:
This is a good entry point for a review. Essentially, supergroups like 'announcements' used to be a part of `@documents` but have been moved to `@supergroups`. Having two presenters is intended to make things simpler.

Reply:
Example page: https://govuk-collections-pr-1008.herokuapp.com/government/organisations/department-for-environment-food-rural-affairs

Versus production: https://www.gov.uk/government/organisations/department-for-environment-food-rural-affairs

Reply:
@bilbof I'd add these comments to the commit message so they don't get lost.